src/validators/float.rs

```diff
@@ -33,6 +34,7 @@ impl BuildValidator for FloatValidator {
         } else {
             Ok(Self {
                 strict: is_strict(schema, config)?,
+                allow_inf_nan: schema.get_as(intern!(py, "allow_inf_nan"))?.unwrap_or(true),
```

Comment:
maybe we should use `schema_or_config(schema, config, "allow_inf_nan", "float_allow_inf_nan")` so this can be set on config too?

Reply:
I agree. Feels a bit like "strict" where I expect people to mostly use them directly at config level. On it

Reply:
ed564508cb23bc58f068703f78c1202397129a33